modules/ROOT/pages/amazon/amazon-sqs-connector-examples.adoc

```diff
@@ -214,6 +226,7 @@ image::amazon/amazon-sqs-receive-message.png[receive-message]
 |===
 
 == Demo Mule Application XML Code
+// I WONDER IF WE SHOULD NAME THIS "EXAMPLE" CODE RATHER THAN "DEMO" CODE, to avoid confusion. I also wonder if this should not be included BEFORE the steps of the example that uses it? No mention has been made of it as a prerequisite; it could join the other prerequisite identified above.  //
```

Comment:
```suggestion
```

Reply:
I WONDER IF WE SHOULD NAME THIS "EXAMPLE" CODE RATHER THAN "DEMO" CODE, to avoid confusion. I also wonder if this should not be included BEFORE the steps of the example that uses it? No mention has been made of it as a prerequisite; it could join the other prerequisite identified above.

Reply:
Changed to "Exam[ple Code." I don't agree with this. The XML code is not a prerequisite. It is a result of following the previous steps.